ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -15,7 +15,7 @@ spec:
   query: |
     SELECT 1 FROM security_profile_info WHERE password_history_size >= 24;
   purpose: Informational
-  tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.1
+  tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.1, CIS_Bullet_1.1.1
```

Comment:
We need to change `CIS-win10-stand-alone-1.1.1` --> `CIS-win10-enterprise-1.1.1` 
Question: What is this `CIS_Bullet_1.1.1` and how is it different from the above?

Reply:
Applies to all items below as well.

Reply:
Done